audiomentations/augmentations/bit_crush.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+from numpy.typing import NDArray
+
+from audiomentations.core.transforms_interface import BaseWaveformTransform
+
+class BitCrush(BaseWaveformTransform):
+    """
+    Apply a bit crush effect the audio by reducing of the resolution or bandwidth of audio 
+    data. This class implements bit depth reduction, which reduces the number of bits 
+    that can be used for representing each audio sample.
+    """
+    
+    supports_multichannel = True
+    
+    def __init__(self, bit_depth: int = 4, p: float = 0.5):
```

Comment:
In the context of data augmentation, it would be nice if the class can randomize the bit depth parameter, like this:
```suggestion
    def __init__(self, min_bit_depth: int = 5, max_bit_depth: int = 10, p: float = 0.5):
```

Can you add this functionality?

Reply:
Sure, I'll do it.

Reply:
Done